### PR TITLE
My hw6

### PR DESCRIPTION
**Pull request by OlehHalush**

### CODE REVIEW

File: src/main/java/hw6/HW6_Task6.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        InputStreamReader r = new InputStreamReader(System.in);
+        BufferedReader br = new BufferedReader(r);
+        String number = "";
+        try {
+            while (!number.equals("any string")) {
+                System.out.println("Enter some number:");
+                myLisOfInts.add(Integer.parseInt(number = br.readLine()));
+                System.out.println("data is: " + number);
+            }
+        } catch (IOException ex) {
```
Comment (yuriy-khortashko) [inline src/main/java/hw6/HW6_Task6.java]: You can use one catch block, as both exceptions are handled in the same way, like:
```
        } catch (IOException | NumberFormatException ex) {
            for (int i : myLisOfInts) {
                System.out.println(i);
            }
        } 
```

File: src/main/java/hw6/HW6_Task6.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        readData();
+    }
+
+    public static void readData() throws IOException {
+        List<Integer> myLisOfInts = new ArrayList<>();
+        InputStreamReader r = new InputStreamReader(System.in);
+        BufferedReader br = new BufferedReader(r);
+        String number = "";
+        try {
+            while (!number.equals("any string")) {
```
Comment (yuriy-khortashko) [inline src/main/java/hw6/HW6_Task6.java]: Condition is always false, if you enter "any string", you will get an exception on line 29

File: src/main/java/hw6/HW6_Task6.java (hunk 1 above)
Comment (OlehHalush) [reply src/main/java/hw6/HW6_Task6.java]: Done

File: src/main/java/hw6/HW6_Task6.java (hunk 2 above)
Comment (OlehHalush) [reply src/main/java/hw6/HW6_Task6.java]: Done

**Pull request closed without merging**